config/locales/devise.en.yml

```diff
@@ -0,0 +1,65 @@
+# Additional translations at https://github.com/heartcombo/devise/wiki/I18n
```

Comment:
Again, this file is generated by `rails generate devise:install` and I've not modified it further.

I'll update the views at some point soon so that all the copy within markups is defined in translation files as it's usually considered best practice.